src/Cli.Tests/EndToEndTests.cs

```diff
@@ -682,24 +679,25 @@ public void TestHelpWriterOutput(string command, string flags, string[] expected
         process.Kill();
     }
 
-    [DataRow("", "--version", DisplayName = "Checking dab version with --version.")]
-    [DataTestMethod]
-    public void TestVersionHasBuildHash(
-        string command,
-        string options
-    )
+    /// <summary>
+    /// When CLI is started via: dab --version, it should print the version number
+    /// which includes the commit hash. For example:
```

Comment:
We have an issue that contradicts this test: according to the issue, the commit hash should NOT be included in the version: https://github.com/Azure/data-api-builder/issues/1456 that is displayed via CLI. 

Can we get clarity on this? to remove the hash altogether or keep it in CLI but not in appname?

Reply:
ah per that issue, the problem was that we were relying on FileVersionInfo.ProductVersion which included the hash. We can instead concatenate the following to enable returning a version without the hash and still allowing the hash value to be present in the assembly metadata for NuGet usage.

```csharp
fileVersionInfo.ProductMajorPart + "." + fileVersionInfo.ProductMinorPart + "." + fileVersionInfo.ProductBuildPart;
```

Reply:
I need to keep this for CLI. We can have follow up item track reimplementing the health screen for this limitation: https://github.com/commandlineparser/commandline/issues/757#issuecomment-1005505288

Reply:
the follow up item and existing issue is the one you linked: https://github.com/Azure/data-api-builder/issues/1456. I'll add my suggestions for how to implement this there.